components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -3868,6 +3868,8 @@ components:
         policyId:
           type: string
           example: RBAC
+        type:
+          type: string
```

Comment:
shall we add an example/ list of examples

Reply:
Added